public/javascripts/notebook/observable.js

```diff
@@ -1,5 +1,29 @@
-define(['jquery', 'base/js/events', 'knockout', 'equiv'], function ($, events, ko, equiv) {
+define(['jquery', 'base/js/events', 'knockout', 'equiv', 'underscore'], function ($, events, ko, equiv, _) {
+ko.extenders.async = function(computedDeferred, initialValue) {
```

Comment:
looks like this is based on:
https://github.com/knockout/knockout/wiki/asynchronous-dependent-observables
or https://smellegantcode.wordpress.com/2012/12/10/asynchronous-computed-observables-in-knockout-js/
add a link and explanation maybe?